src/main/java/sx/blah/discord/handle/obj/IUser.java

```diff
@@ -153,4 +153,17 @@
 	 * @return True if muted, false if otherwise.
 	 */
 	boolean isMuted(IGuild guild);
+
+	/**
+	 * Gets whether this user is deafened locally.
+	 *
+	 * @return True if deafened, false if otherwise.
+     */
+	boolean isDeafLocally();
```

Comment:
"Gets whether this user is deafened locally (meaning they deafened themselves)."
something like that?